src/components/Header/HeaderUserMenu/HeaderUserMenu.test.js

```diff
@@ -71,4 +77,26 @@ describe('Header user menu', () => {
     );
     expect(getByText('Upgrade')).toBeInTheDocument();
   });
+
+  it('user have SMS enabled', () => {
+    const { getByText } = render(
+      <IntlProvider>
+        <HeaderUserMenu user={userData} />
+      </IntlProvider>,
+    );
+    expect(getByText('Available for SMS')).toBeInTheDocument();
+  });
+
+  it('user have SMS disabled', () => {
+    userData.sms = {
+      remainingCredits: '',
+    };
+
+    const { queryByText } = render(
+      <IntlProvider>
+        <HeaderUserMenu user={userData} />
+      </IntlProvider>,
+    );
+    expect(queryByText('Available for SMS')).not.toBeInTheDocument();
```

Comment:
```suggestion
    expect(queryByText(userData.sms.description)).not.toBeInTheDocument();
```

Reply:
In this case `userData.sms.description` doesn't have any value. I want to verify that there's no any element with this specific text.

Reply:
In which way do you think we can avoid this hardcoded value and still validate?